locales/sk/json.json

```diff
@@ -584,30 +584,30 @@
     "Sao Tome and Principe": "Svätý Tomáš a Princov ostrov",
     "Saudi Arabia": "Saudská Arábia",
     "Save": "Uložiť",
-    "Saved.": "Uložený.",
+    "Saved.": "Uložené.",
     "Scroll to bottom": "Prejdite nadol",
     "Scroll to top": "Prejdite na začiatok",
     "Search": "Vyhľadávanie",
     "Select": "Vyberte",
     "Select a different plan": "Vyberte iný plán",
     "Select a log file...": "Vyberte súbor denníka...",
-    "Select A New Photo": "Vyberte Novú Fotografiu",
-    "Select Action": "Vyberte Akciu",
-    "Select all": "Vybrať Všetko",
-    "Select All Dropdown": "Vyberte rozbaľovaciu ponuku Všetky",
+    "Select A New Photo": "Vyberte novú fotografiu",
+    "Select Action": "Vyberte akciu",
+    "Select all": "Vyberte všetko",
+    "Select All Dropdown": "Vyberte všetko",
     "Select this page": "Vyberte túto stránku",
-    "Send Password Reset Link": "Poslať odkaz pre obnovenie hesla",
+    "Send Password Reset Link": "Odoslať odkaz na obnovu hesla",
     "Senegal": "Senegal",
-    "September": "septembra",
+    "September": "September",
```

Comment:
The value also needs to be added to the `_excludes.json` file, otherwise the status will show up as requiring translation.

Reply:
Added